src/omnisharp/OmnisharpPackageCreator.ts

```diff
@@ -3,24 +3,28 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { Package } from "../packageManager/Package";
+import { PackageJSONPackage } from "../packageManager/PackageJSONPackage";
```

Comment:
There should be exactly one component that knows/cares what a PackageJSONPackage is. Why do we need to look at the Package.json in two places?